dds/src/dds/topic_definition/type_support.rs

```diff
@@ -10,15 +10,23 @@ use crate::{
 
 pub use dust_dds_derive::{DdsGetKey, DdsHasKey, DdsRepresentation, DdsSetKeyFields, DdsType};
 
-pub type RepresentationType = [u8; 2];
```

Comment:
The representation type is defined as [u8; 2]. This should be kept, to show the intent clearly.
It should stay as such.

Reply:
I put the RepresentationType back